w3af/core/controllers/dependency_check/platforms/current_platform.py

```diff
@@ -30,9 +30,10 @@
 from .openbsd import OpenBSD5
 from .suse import SuSE
 from .default import DefaultPlatform
+from .linuxMint17 import LinuxMint17
```

Comment:
File name should be either:
 * All lower case `linuxmint17`
 * Underscore separated `linux_mint_17`

camelCase is not accepted